### DISCUSSION TITLE
Why does LocalResource lack a `refetch` method?

### DISCUSSION THREAD

**Discussion by linw1995** [leptos-rs/leptos#3381] [Q&A] [answered]
When updating from v0.6.x to v0.7.x, this issue was discovered. `Resource` has a `refetch` method but requires `Send + Sync + 'static`. `LocalResource` doesn’t have these requirements but lacks the `refetch` method.

It's unclear whether this is a bug or an intentional design choice. Hopefully, someone kind-hearted can provide an explanation. 🙏

**Comment by gbj** [comment, marked as answer]
You are correct that this method does not exist.

You can replicate it pretty easily.

All it does is create an `RwSignal<usize>`, and track it in the body; calling `.refetch()` increments the signal.
